stories/SQForm.stories.js

```diff
@@ -333,6 +348,40 @@ export const formWithCheckboxGroup = () => {
   );
 };
 
+export const basicFormWithMultiSelect = () => {
```

Comment:
Any chance you could add a few more fields or something to make the card a little bigger? I say that only because of the way the options move up and down in the loom video. It's kind of janky, and I wonder if it's a product of showing it in storybook, or if it's behave that way in the wild. Is there any way to address that? I'd lay money that Nemeth will balk at that.

Reply:
I wondered the exact same thing while poking around 💯 . I believe it's a problem with storybook tbh. I'll spend a few minutes poking. But, I could take the same code and place it in Life CRM and wasn't getting that jank?